tests/lib/rules/no-console.js

```diff
@@ -32,7 +32,15 @@ ruleTester.run("no-console", rule, {
         { code: "console.info(foo)", options: [ { allow: ["warn", "info"] } ] },
         { code: "console.warn(foo)", options: [ { allow: ["error", "warn"] } ] },
         { code: "console.error(foo)", options: [ { allow: ["log", "error"] } ] },
-        { code: "console.log(foo)", options: [ { allow: ["info", "log", "warn"] } ] }
+        { code: "console.log(foo)", options: [ { allow: ["info", "log", "warn"] } ] },
+
+        // disallowing globals
+        { code: "window.console.info(foo)", options: [ { allow: ["info"], disallowGlobals: ["self"] } ] },
+        { code: "self.console.error(foo)", options: [ { allow: ["error"], disallowGlobals: ["window"] } ] },
```

Comment:
Can you check something like `foo.window.console.log`? Just want to make sure deeply nested references don't trigger a warning.